sdks/go/pkg/beam/pardo.go

```diff
@@ -428,31 +428,20 @@ func ParDoErrorFormatter(doFn interface{}, parDo interface{}) string {
 	parDoName := reflectx.FunctionName(parDo)
 	parDoOutSize := reflectx.FunctionOutputSize(parDo)
 
-	useParDo := reflectx.FunctionName(RecommendParDo(doFnOutSize))
+	useParDo := reflectx.FunctionName(recommendParDo(doFnOutSize))
```

Comment:
Note that now that you've changed the return type of `recommendParDo`, you don't need to call `reflectx.FunctionName` anymore.

Reply:
45f4a84e57